src/api/middlewares/Authentication.ts

```diff
@@ -63,6 +64,8 @@ export async function Authentication(
 	if (!req.headers.authorization)
 		return next(new HTTPError("Missing Authorization Header", 401));
 
+	Sentry.setUser({ id: req.user_id });
```

Comment:
Has this been tested for large loads? Iirc Sentry doesn't handle async functions very well. See: https://github.com/getsentry/sentry-javascript/issues/4071

Reply:
It has not, but should be fine, as it only sets the current user of the current request, accuracy of said data is not nescesary, as it is only used to get a rough idea of how commonly an endpoint is used, besides hitcounts.

Reply:
Hm it would be nice if it did work correctly for debugging purposes, for example if a user is corrupted somehow. But yeah you're right